mne/viz/tests/test_ica.py

```diff
@@ -89,6 +91,19 @@ def test_plot_ica_sources():
     epochs.info['bads'] = []
     with warnings.catch_warnings(record=True):  # no labeled objects mpl
         ica.plot_sources(epochs.average())
+        evoked = epochs.average()
+        fig = ica.plot_sources(evoked)
+        # Test a click
+        ax = fig.get_axes()[0]
+        line = ax.lines[0]
+        _fake_click(fig, ax,
+                    [line.get_xdata()[0], line.get_ydata()[0]], 'data')
+        _fake_click(fig, ax,
+                    [ax.get_xlim()[0], ax.get_ylim()[1]], 'data')
+        # plot with bad channels excluded
+        ica.plot_sources(evoked, exclude=[0])
+        ica.exclude = [0]
+        ica.plot_sources(evoked)  # does the same thing
```

Comment:
For this PR I'd probably do `nosetests mne/viz/tests/test_ica --with-coverage` and then look for `mne.viz.ica` and try to get coverage as high as possible, even increasing coverage on lines I haven't changed. You can also use coveralls reports, but then you have to wait ~20 minutes for Travis, so it's useful for an initial or final check but not really iterating:

https://coveralls.io/builds/2688124/source?filename=mne%2Fviz%2Fica.py